components/Layout.tsx

```diff
@@ -6,6 +6,7 @@ import grid from '@/styles/grid.module.css';
 import heading from '@/styles/headings.module.css';
 import list from '@/styles/List.module.css';
 import Link from 'next/link'
+import Card from './Card'
```

Comment:
This is causing a build issue since there is no component named `Card`.
I think you are referring to the `card` component. Check the previous comment